imagedephi/redact/svs.py

```diff
@@ -204,12 +209,25 @@ def report_plan(self) -> dict[str, dict[str | int, str | int]]:
                     rule = self.description_redaction_steps[key_name]
                     operation = self.determine_redaction_operation(rule, image_description)
                     logger.info(f"SVS Image Description - {key_name}: {operation}")
-                    report[self.image_path.name][key_name] = operation
+                    report[self.image_path.name][key_name] = {"action": operation, "value": _data}
                 continue
             rule = self.metadata_redaction_steps[tag.value]
             operation = self.determine_redaction_operation(rule, ifd)
             logger.info(f"Tiff Tag {tag.value} - {rule.key_name}: {operation}")
-            report[self.image_path.name][rule.key_name] = operation
+            if ifd["tags"][tag.value]["datatype"] == 7:
```

Comment:
Rather than `7`, this should be something like `tifftools.Datatype.UNDEFINED.value`

Reply:
Fixed in 02dac497951c541ca3e3ac007eafd1d3c6e84882